docs/development_suite/deploy/gitops-based/index.md

```diff
@@ -4,10 +4,6 @@ title: GitOps deployment strategy
 sidebar_label: GitOps deployment strategy
 ---
 
-:::caution
```

Comment:
I think the gitops support should still be considered a beta feature @davidebianchi what do you think?

Reply:
nah, changed my mind. Let's remove the beta imho, we have support for gitops, not for a specific tool but that does not configure the feature as beta